Source/GameBaseFramework/Private/Characters/GBFSimpleCharacterWithAbilities.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include "GAS/Abilities/GBFAbilitySet.h"
 #include "GAS/Components/GBFAbilitySystemComponent.h"
+#include "GBFTags.h"
+#include "GameFramework/CharacterMovementComponent.h"
```

Comment:
<>

Reply:
fixed

Reply:
GBFTags should not have <>, but just keep ""

Reply:
![image](https://github.com/user-attachments/assets/8ce81fd9-070b-4f2e-a518-8ecad84a7c89)
not modified

Reply:
done.